tests/test_distro.py

```diff
@@ -122,23 +123,26 @@ class DistroTestCase(object):
     represented in the `DISTROS` subtree.
     """
 
-    def setup_method(self, test_method):
+    def setup_method(self, _):
```

Comment:
With my picky hat on: No objections but my vote for moving to commit dedicated to unused variables.

Reply:
Reverted.